### DISCUSSION TITLE
Import global constants from Python in a Rust file

### DISCUSSION THREAD

**Discussion by JackAshwell11** [PyO3/pyo3#2812] [Questions]
So if I have a Python file `const.py` (assuming this is a placeholder for more complicated data types):
```py
VAR1 = 5
VAR2 = "Temp"
```
How can I access and use these variables in a Rust file?

**Comment by birkenfeld** [comment]
You'd import the file (assuming it's accessible as a module) using `PyModule::import`. Then you can access its members as a `&PyAny` by `getattr`.

To get to a more concrete type, you can either `extract` (to a Rust native type) or `downcast` (to e.g. `&PyString`).
